docs/content/tutorials/helm/index.md

```diff
@@ -0,0 +1,652 @@
+pod/webapp-76db7964d8-plc2s                  1/1     Running   0          37s
+
+NAME                          TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)    AGE
+service/redis-r5tcrra3d7uh6   ClusterIP   10.43.104.63   <none>        6379/TCP   20m
+
+NAME                                  READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/redis-r5tcrra3d7uh6   1/1     1            1           20m
+deployment.apps/webapp                  1/1     1            1           20m
+
+NAME                                             DESIRED   CURRENT   READY   AGE
+replicaset.apps/redis-r5tcrra3d7uh6-7bcd8b8d8d   1         1         1       20m
+replicaset.apps/webapp-79d5dfb99                   0         0         0       20m
+replicaset.apps/webapp-76db7964d8                  1         1         1       37s
+replicaset.apps/webapp-687dcf5cdf                  0         0         0       38s
+
+NAME                TYPE                                  SECRET   STATUS
+recipe.radapp.io/db   Applications.Datastores/redisCaches            Ready
+```
+
+Depending on the timing you may see pods in the `Terminating` state. This is normal as old replicas take some time to shut down.
+
+Check the Radius status again. Now Radius is aware of the connection from `webapp->db`:
+
+```bash
+rad app graph -a demo -g default-demo
+```
+
+The output should look like the example below:
+
+```txt
+> rad app graph -a demo -g default-demo
+Displaying application: demo
+
+Name: webapp (Applications.Core/containers)
+Connections:
+  webapp -> db (Applications.Datastores/redisCaches)
+Resources:
+  webapp (kubernetes: apps/Deployment)
+
+Name: db (Applications.Datastores/redisCaches)
+Connections:
+  webapp (Applications.Core/containers) -> db
+Resources:
+  redis-r5tcrra3d7uh6 (kubernetes: apps/Deployment)
+  redis-r5tcrra3d7uh6 (kubernetes: core/Service)
+
+
+```
+
+## Step 7. Try it out
+
+In this step you can access the application and explore its features. Since the container is running inside Kubernetes, you need to run a port-forward to use it locally.
+
+Run:
+
+```bash
+kubectl port-forward -n demo deployment/webapp 3000
+```
+
+If you are inside Codespaces this will open a new browser tab that you can use to access the webapp. 
+
+If you are not using Codespaces then open your browser and navigate to `http://localhost:3000`
+
+<img src="./demo-with-redis-screenshot.png" alt="Screenshot of the demo container" width=500>
+<br /><br />
+
+Congrats! You're running your first Radius app.
+
+You can use the homepage to view information about the container and its settings.
+
+Navigate to the TODO List tab and test out the application. Using the TODO page will update the saved state in Redis.
+
+<img src="./demo-with-todolist.png" alt="Screenshot of the todolist" width=500>
+<br /><br />
+
+When you're ready to move on to the next step, use `CTRL+C` to exit the command.
+
+## Cleanup and next steps
+
+To delete your app, run the following command:
```

Comment:
We should also add a note on how this also cleans up all resources created as part of Recipes

Reply:
done